ogzaf/demographics.py

```diff
@@ -479,8 +671,7 @@ def get_pop_objs(E, S, T, min_yr, max_yr, curr_year, GraphDiag=False):
                 path, length T + S
 
     """
-    assert curr_year >= 2011
-    # age_per = np.linspace(min_yr, max_yr, E+S)
+    assert curr_year >= 2021
     fert_rates = get_fert(E + S, min_yr, max_yr, graph=False)
     mort_rates, infmort_rate = get_mort(E + S, min_yr, max_yr, graph=False)
```

Comment:
```suggestion
    fert_rates = get_fert(min_yr, max_yr, graph=False)
    mort_rates, infmort_rate = get_mort(min_yr, max_yr, graph=False)
```

Reply:
@SeaCelo. I need to work on this a little bit. We discussed hard coding the `totpers` object to be `100` and `min_yr=1` and `max_yr=100`. But I need to look back at all the tests and `calibrate.py` that use this to make sure this is OK. We might need it to be flexible.

Reply:
@SeaCelo. I removed the `min_yr` and `max_yr` arguments in `demographics.py` and `calibrate.py` as they are hard coded to `min_yr=1` and `max_yr=100` because the UN population data have annual ages. I left the `totpers` argument because we need to be able to transform the annual data to model periods that are bigger than one year (`totpers<100`).